docs/using_color.md

```diff
@@ -0,0 +1,141 @@
+
+````js
+var UI = require('ui');
+var Feature = require('platform/feature');
+
+var card = new UI.Card({
+  title: 'Using Color',
+  titleColor: Feature.color('sunset-orange', 'black'),
+  subtitle: 'Readability',
+  subtitleColor: Feature.color('#00dd00', 'black'),
+  body: 'Contrast',
+  bodyColor: Feature.color(0x9a0036, 'black'),
+  backgroundColor: Feature.color('light-gray', 'white'),
+});
+
+card.show();
+````
+
+Whether you have a color Pebble or not, you will want to test your app in all platforms. You can see how your app looks in multiple platforms with the following local SDK command.
+
+> `pebble build && pebble install --emulator=aplite && pebble install --emulator=basalt && pebble install --emulator=chalk`
+
+Using too much color such as in the previous example can be overwhelming however. Just using one color that stands out in a single place can have a more defined effect and remain readable.
+
+````js
+var card = new UI.Card({
+  status: {
+    color: 'white',
+    backgroundColor: Feature.color('electric-ultramarine', 'black'),
+    separator: 'none',
+  },
+  title: 'Using Color',
+  subtitle: 'Readability',
+  body: 'Contrast',
+});
+````
+
+Likewise, if introducing an action bar, you can remove all color from the status bar and instead apply color to the action bar.
+
+````js
+var card = new UI.Card({
+  status: true,
+  action: {
+    backgroundColor: Feature.color('jazzberry-jam', 'black'),
+  },
+  title: 'Dialog',
+  subtitle: 'Action',
+  body: 'Button',
+});
+````
+
+When changing the background color, note that the status bar also needs its background color changed too if you would like it to match.
+
+````js
+var backgroundColor = Feature.color('light-gray', 'black');
+var card = new UI.Card({
+  status: {
+    backgroundColor: backgroundColor,
+    separator: Feature.round('none', 'dotted'),
+  },
+  action: {
+    backgroundColor: 'black',
+  },
+  title: 'Music',
+  titleColor: Feature.color('orange', 'black'),
+  subtitle: 'Playing',
+  body: 'Current Track',
+  backgroundColor: backgroundColor,
+});
+````
+
+For a menu, following this style of coloring, you would only set the `highlightBackgroundColor`.
+
+````js
+var menu = new UI.Menu({
+  status: {
+    separator: Feature.round('none', 'dotted'),
+  },
+  highlightBackgroundColor: Feature.color('vivid-violet', 'black'),
+  sections: [{
+    items: [{ title: 'One', subtitle: 'Using Color' },
+            { title: 'Color', subtitle: 'Color Formats' },
+            { title: 'Hightlight', subtitle: 'Readability' }],
+  }],
+});
+
+menu.show();
+````
+
+In the examples above, mostly black text on white or light gray is used which has the most contrast. Try to maintain this amount of contrast with text. Using dark gray on light gray for example can be unreadable at certain angles in the sunlight or in darkly lit areas.
+
+Well, that was a lot! [Feature detection](feature_detection.md) coming up next.
```

Comment:
```suggestion
<breadcrumb>
```

Reply:
Oh ok, so that's how those work. Might as well add em to all the pages